Source/Libraries/GSF.Web/Model/ModelController.cs

```diff
@@ -378,7 +378,7 @@ public virtual IHttpActionResult Delete(T record)
         [HttpPost, Route("SearchableList")]
         public virtual IHttpActionResult GetSearchableList([FromBody] PostData postData)
         {
-            if (GetAuthCheck() && !AllowSearch)
+            if (!GetAuthCheck() && !AllowSearch)
```

Comment:
That Logic will require a user to be authorized for `GET` OR `AllowSearch` to be true for it to continue since either `GetAuthCheck()` or `AllowSerach` being `true` will cause `Unauthorized`

The idea here was the user needs to be authorized for GET AND the model has to bel searchable. 

```suggestion
            if (!GetAuthCheck() || !AllowSearch)
```

Reply:
Note that the issue you encountered should probably be resolved by adding `AllowSearch` to the model since that would have made the old logic work properly. There is still an issue with the old logic approving users without `GET` access tho